js/bw.js

```diff
@@ -292,6 +292,8 @@ module.exports = class bw extends Exchange {
                 'info': currency,
                 'name': code,
                 'active': active,
+                'deposit': undefined,
```

Comment:
I think we should use `rechargeFlag` to determine if `deposit` is `true` or `false`.

Reply:
Should we also change active to this?
```JS
const active = deposit && withdraw &&  state === 1;
```

Reply:
I would keep it as is for now.